Python/dynamic/parameters.py

```diff
@@ -93,40 +189,62 @@ def get_full_parameters():
     ltilde = 1.0
     g_y_annual = 0.03
     g_y = (1 + g_y_annual)**(float(ending_age-starting_age)/S) - 1
-    # TPI parameters
-    maxiter = 250
-    mindist_SS = 1e-9
-    mindist_TPI = 1e-6
-    nu = .40
-    # Ellipse parameters
+    #   Ellipse parameters
     b_ellipse = 25.6594
     k_ellipse = -26.4902
     upsilon = 3.0542
+
     # Tax parameters:
+    #   Income Tax Parameters
     mean_income_data = 84377.0
     a_tax_income = 3.03452713268985e-06
     b_tax_income = .222
     c_tax_income = 133261.0
     d_tax_income = .219
-    retire = np.round(9.0 * S / 16.0) - 1
-    # Wealth tax params
-    # These won't be used for the wealth tax, h and m just need
-    # need to be nonzero to avoid errors
+    #   Wealth tax params
+    #       These are non-calibrated values, h and m just need
+    #       need to be nonzero to avoid errors. When p_wealth
+    #       is zero, there is no wealth tax.
     h_wealth = 0.1
     m_wealth = 1.0
     p_wealth = 0.0
-    # Tax parameters that are zeroed out for SS
-    # Initial taxes below
+    #   Bequest and Payroll Taxes
     tau_bq = np.zeros(J)
     tau_payroll = 0.15
-    # Flag to prevent graphing from occuring in demographic, income, wealth, and labor files
+    retire = np.round(9.0 * S / 16.0) - 1
+
+    # Simulation Parameters
+    MINIMIZER_TOL = 1e-14
+    MINIMIZER_OPTIONS = None
+    PLOT_TPI = True
+    maxiter = 250
+    mindist_SS = 1e-9
+    mindist_TPI = 1e-6
+    nu = .4
     flag_graphs = False
+    #   Calibration parameters
+    # These guesses are close to the calibrated values
+    chi_b_guess = np.array([2, 10, 90, 350, 1700, 22000, 120000])
+    chi_n_guess = np.array([47.12000874 , 22.22762421 , 14.34842241 , 10.67954008 ,  8.41097278
```

Comment:
Oh nevermind my comment above. I see. Yes this is a good place for these parameters.